cmd/etcd-launcher/main.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/minio/minio-go"
```

Comment:
Please  fix imports also in other places. They should be in order:
- golang
- other 
- kubermatic
- k8s